### PR TITLE
Refactor: functional tests and SQL statements with uppercase SQL keywords

### PR DESCRIPTION
**Pull request by souravroy2021**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDCC-3459

### Change description ###

Refactored functional tests and SQL statements with uppercase SQL keywords

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by amitkumarshrivastava — APPROVED** [review]

File: src/functionalTest/resources/application-integration.yml [hunk 1]
```diff
@@ -93,12 +95,12 @@ route:
     #Personal-Test only for testing else -Test not needed by main code
     parent-name: Personal-Test
     truncate-sql:
-      sql:truncate table judicial_office_authorisation restart identity?dataSource=dataSource
+      sql:TRUNCATE table judicial_office_authorisation RESTART IDENTITY?dataSource=dataSource
```
Comment (shreedhar-hmcts) [inline src/functionalTest/resources/application-integration.yml:98]: table needs to be in CAPITAL ?

File: src/integrationTest/resources/application-integration-test.yml [hunk 2]
```diff
@@ -81,40 +81,40 @@ route:
     parent-failure-enabled: true
     parent-name: Personal
     truncate-sql:
-      sql:truncate table judicial_office_authorisation restart identity?dataSource=dataSource
+      sql:TRUNCATE table judicial_office_authorisation RESTART IDENTITY?dataSource=dataSource
```
Comment (shreedhar-hmcts) [inline src/integrationTest/resources/application-integration-test.yml:84]: table required to be in caps ?

File: src/main/resources/application-jrd-router-orchestartion.yaml [hunk 3]
```diff
@@ -50,12 +50,12 @@ route:
     parent-failure-enabled: true
     parent-name: Personal
     truncate-sql:
-      sql:truncate table judicial_office_authorisation restart identity?dataSource=dataSource
+      sql:TRUNCATE table judicial_office_authorisation RESTART IDENTITY?dataSource=dataSource
```
Comment (shreedhar-hmcts) [inline src/main/resources/application-jrd-router-orchestartion.yaml:53]: same

File: src/functionalTest/resources/application-integration.yml [hunk 4]
```diff
@@ -27,15 +27,17 @@ start-route: direct:JRD
...
 archival-cred: credentials=#credsreg&operation=updateBlockBlob
 archival-date-format: dd-MM-yyyy
 active-blob-path: azure-blob://${azure.storage.account-name}/jud-ref-data
 file-read-time-out: 2000
-scheduler-audit-select: select count(*) from dataload_schedular_audit where date(scheduler_start_time) = current_date
-scheduler-audit-prev-day: select file_name, scheduler_start_time, status from dataload_schedular_audit where date(scheduler_start_time) = current_date - INTEGER '1'
-invalid-exception-sql: insert into dataload_exception_records(table_name,scheduler_start_time,scheduler_name,error_description,updated_timestamp) values(?, ?, ?, ?, ?)
-invalid-jsr-sql: insert into dataload_exception_records(table_name,scheduler_start_time,scheduler_name,key,field_in_error,error_description,updated_timestamp,row_id) values(?, ?, ?, ?, ?, ?, ?, ?)
-scheduler-insert-sql: insert into dataload_schedular_audit (scheduler_name,file_name, scheduler_start_time,scheduler_end_time,status) values(?,?,?,?,?)
+scheduler-audit-select: SELECT count(*) FROM dataload_schedular_audit WHERE date(scheduler_start_time) = current_date
```
Comment (shreedhar-hmcts) [inline src/functionalTest/resources/application-integration.yml:36]: current_date should be in  CAPS ?

File: src/functionalTest/resources/application-integration.yml (hunk 1 above)
Comment (souravroy2021) [reply src/functionalTest/resources/application-integration.yml]: It has been fixed.

File: src/integrationTest/resources/application-integration-test.yml (hunk 2 above)
Comment (souravroy2021) [reply src/integrationTest/resources/application-integration-test.yml]: It has been fixed.

File: src/main/resources/application-jrd-router-orchestartion.yaml (hunk 3 above)
Comment (souravroy2021) [reply src/main/resources/application-jrd-router-orchestartion.yaml]: It has been fixed.

File: src/functionalTest/resources/application-integration.yml (hunk 4 above)
Comment (souravroy2021) [reply src/functionalTest/resources/application-integration.yml]: It has been fixed.

**Review by shreedhar-hmcts — APPROVED** [review]

**Pull request merged**